packages/genesys-spark-tokens/data/tokens.json

```diff
@@ -7436,6 +7436,12 @@
           }
         }
       }
+    },
+    "cta_group": {
```

Comment:
If you're adding new tokens, then you'll need to generate the token snapshots on your local too. 

Checkout the branch and then run the 'npm run new-tokens-data'. Commit the updated files to your branch.

Reply:
Added snapshots :)